integrations/acquisition/covidcast/test_is_latest_issue.py

```diff
@@ -68,17 +67,17 @@ def test_signal_latest(self):
       CovidcastRow('src', 'sig', 'day', 'state', 20200414, 'pa', 
                    1.5, 2.5, 3.5, nmv, nmv, nmv, 20200414, 0)
     ]
-    self._db.insert_or_update_bulk(rows)
+    self._db.insert_or_update_batch(rows)
     self._db.run_dbjobs()
     self._db._cursor.execute(self.viewSignalHistory)
-    totalRows = len(list(self._db._cursor.fetchall()))
+    self.totalRows = len(list(self._db._cursor.fetchall()))
 
     #sanity check for adding dummy data
     sql = f'SELECT `issue` FROM {Database.latest_table} where `time_value` = 20200414'
     self._db._cursor.execute(sql)
     record = self._db._cursor.fetchall()
     self.assertEqual(record[0][0], 20200414)
-    self.assertEqual(len(record), 1) #check 1 entry
+    self.assertEqual(len(record), 1) #check 1 entry present
```

Comment:
line comments should add context for future readers, beyond just restating the code -- for example, this modification explains why we only expect 1 entry for this query: 
```suggestion
    self.assertEqual(len(record), 1) #placeholder data only has one issue for 20200414
```

Reply:
I edited them accordingly, let me know if they showed up!